### DISCUSSION TITLE
Can't enter Dungeon: Debugger

### DISCUSSION THREAD

**Discussion by barthon-b** [ByteLegend/ByteLegend#106] [General]
I can't seem to be able to enter `Dungeon: Debugger` and it seems empty anyway. Is this by design or is it just me doing something wrong?

**Comment by blindpirate** [comment]
Here is what I see right now: 

![image](https://user-images.githubusercontent.com/12689835/147008184-04ebbd22-64b4-453a-93d0-e611e07ac375.png)

Do you mean it's broken or there's nothing? It's empty right now because I haven't created any missions for it.

**Comment by barthon-b** [comment]
Hi, yes - it's empty as you have shown but my character cannot enter it either. That said, please ignore me - I just realized the game is not finished yet so these type of issues are normal. Thanks for your continued effort, once finished this will be an amazing educational tool. It's an option I was really missing among all the other coding portals, video lectures, and reading. I'd gladly pay a monthly fee to access the material. Have you thought about applying for some sort of a monetary grant from any of the FAANG companies?

**Comment by blindpirate** [comment]
@barthon-b I see your point, yes, there was something wrong with the map, and I've already fixed them. I also put two challenges in debugger dungeon. PTAL!

> Have you thought about applying for some sort of a monetary grant from any of the FAANG companies?

TBH, not yet, the current plan is to continue 1 or 2 months to finish most of features in my mind, at that time I might consider next step.
